lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -114,3 +115,34 @@ def test_anonymous_user_xml(self):
         resp = self.client.get(url)
         self.assertEqual(resp.status_code, 200)
         self.assertNotIn(self.xml_data, resp.content)
+
+
+@attr('shard_1')
+@override_settings(FEATURES=dict(settings.FEATURES, EMBARGO=False))
```

Comment:
I have a PR hanging around that turns off all caches for testing (and then expects tests to turn them on specifically to test behavior with caching). I'd love to get it merged, but haven't had time, yet. It would make this sort of instability much less prevalent.